e2e/test/scenarios/embedding/embed-resource-downloads.cy.spec.ts

```diff
@@ -76,6 +84,12 @@ describeEE(
         cy.get("header").findByText("Export as PDF").click();
 
         cy.verifyDownload("Orders in a dashboard.pdf");
+
+        expectGoodSnowplowEvent({
+          event: "dashboard_pdf_exported",
+          dashboard_id: 0,
```

Comment:
we don't have `dashboard_id` when exporting PDF in static dashboards?

Reply:
I resolved this as we're addressing the same thing in https://github.com/metabase/metabase/pull/46335/files#r1701481340